reana_server/config.py

```diff
@@ -300,9 +300,14 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 
 # Workspace retention rules
 # ==================
-
-WORKSPACE_RETENTION_PERIOD = int(os.getenv("WORKSPACE_RETENTION_PERIOD", "365"))
-"""Maximum allowed period for workspace retention rules."""
+_workspace_retention_period_env = os.getenv("WORKSPACE_RETENTION_PERIOD", "365")
+if _workspace_retention_period_env == "forever":
```

Comment:
Whether it should be `forever` or something else is open to discussing. I am okay with "forever", "keep-forever" or anything else.